### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by OverSx**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @ChugaevMark

### CODE REVIEW

File: ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+	class MainClass
+	{
+		public static void Main(string[] args)
+		{
+			string tc = Console.ReadLine();
+			if (tc.Length <= 10)
```
Comment (OverSx) [inline ConsoleApp1/Program.cs:10]: Некорректно поставлено условие ограничения для данной задачи. В принципе, в данном задании его можно вообще опустить. Либо проверять уже само введенное число на какой-либо заданный диапазон значений.

File: ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+			{
+				int TC = int.Parse(tc);
```
Comment (OverSx) [inline ConsoleApp1/Program.cs:12]: Из-за типа переменной int, а не double, вычисления будут неверны.

File: ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+	class MainClass
+	{
+		public static void Main(string[] args)
+		{
+			string tc = Console.ReadLine();
```
Comment (OverSx) [inline ConsoleApp1/Program.cs:9]: Неясно, что вы хотите от пользователя. При запуске скрипта, перед пользователем просто черный экран. Необходимо добавить запрос для пользователя в виде сообщения, что вы хотите от него получить.

**Comment by OverSx** [comment]
Исправьте код, чтобы задание было выполнено корректно.

**Comment by d-makarenko** [comment]
Оценка: 4/4

**Pull request merged**